githubaction/0.1.0/README.md

```diff
@@ -0,0 +1,257 @@
+# GitHub Actions via Keptn Webhooks
+
+This integration shows how to invoke GitHub Action workflows leveraging [Keptn's Webhook Service](https://keptn.sh/docs/0.10.x/integrations/webhooks/).  With a GitHub actions integration, you can call existing GitHub workflows from Keptn sequences. 
+
+[WATCH THIS VIDEO TUTORIAL](https://www.youtube.com) to see this in action!
+
+# Prerequisites
+
+## Step 1: Configure GitHub
+
+1. GitHub Access Token
+
+    A GitHub personal access token (PAT) is required to pass within the Keptn webhook authorization header in order for the GitHub API to authenticate the request. Follow these instructions in the [GitHub docs](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token) to create your token and be sure that you have given the token access to the `repo` [scope](https://docs.github.com/en/developers/apps/building-oauth-apps/scopes-for-oauth-apps).
+
+1. GitHub repo with a GitHub Action
+
+    The GitHub repository must have at least one action workflow. If you are not familiar with GitHub, we recommend you review the [GitHub getting started documentation](https://docs.github.com/en/actions/learn-github-actions).
+
+1. GitHub Action configured to support repository dispatch
+
+    Use the GitHub API to trigger a webhook event called [repository_dispatch](https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#repository_dispatch) that will start a GitHub action workflow. 
+  
+    To use repository dispatches, configure your GitHub workflow with the `repository_dispatch` trigger and provide a dispatch name that will match to the value. Within the Keptn webhook payload that is sent to the GitHub action API request, the `event_type` attribute is how GitHub matches the request to the GitHub workflow to run.  
+    
+    Below is an example with a repository dispatch with the event type name of `silent`.
+
+    ```
+    name: webhook silent example
+    on:
+      repository_dispatch:
+        types: 
+          - silent
+    ```
+
+## Step 2: Store GitHub token as a Keptn secret
+
+To secure the GitHub Access Token, a Keptn secret must be created.  To do this, go to the Keptn project  `Uniform page > Secret` and click the `Add Secret` button.  On that form fill the following values:
+  * **Name**: Name such as `github-secret` 
+  * **Scope**: `keptn-webhook-service` 
+  * **Key-value pairs**:  Click the `new key-value pair` button and add these values:
+    * **Key** = Name such as `GITHUBTOKEN`
+    * **Value** = The GitHub Access Token from the step from previous section 
+
+The completed form should look as follows:
+
+<img src="images/createsecret.png" width="50%" height="50%">
+
+## Step 3: Create Keptn project
```

Comment:
Please remove "Step 3" as we can (should) assume that a user knows that. 

And I would like to avoid writing a full tutorial here.

Reply:
removed in https://github.com/keptn-sandbox/artifacthub/pull/34/commits/a0ab4afe44744d3ee98494d4f368c60a5f18bea4